### PR TITLE
feat: Update TTL language to Time until shutdown

### LINKED ISSUE #1784: Feat: rename auto-off TTL in UI to normal English
## What is your suggestion?
"TTL" is too jargony for our UI.  We should call it something more accessible like "expires after" or something like that.

## Why do you want this feature?
User understanding

## Are there any workarounds to get this functionality today?
Nope

## Are you interested in submitting a PR for this?
Eventually

### PR DESCRIPTION
**Pull request by AbhineetJain**
This PR updates the copy of TTL in frontend to Time until shutdown.

## Subtasks

- [x] update the language of the label and helptext.

Fixes #1784 

## Screenshot

image

### CODE REVIEW

File: site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ export const Language = {
   startTimeLabel: "Start time",
   startTimeHelperText: "Your workspace will automatically start at this time.",
   timezoneLabel: "Timezone",
-  ttlLabel: "TTL (hours)",
-  ttlHelperText: "Your workspace will automatically shutdown after the TTL.",
+  ttlLabel: "Time until shutdown (hours)",
```
Comment (kylecarbs) [inline site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:43]: What do you think about `Hours until shutdown`?

Comment (AbhineetJain) [reply site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:43]: There's a comment on the issue that talks about eventually having minutes/seconds resolution as well: https://github.com/coder/coder/issues/1784#issuecomment-1138973156.

**Review by greyscaled — APPROVED** [review]

File: site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx [hunk 2]
```diff
@@ -40,8 +40,8 @@ export const Language = {
...
+  ttlHelperText: "Your workspace will automatically shut down after this much time.",
```
[the remark this answers is not in the document]
Comment (khorne3) [reply site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:44]: Your workspace will automatically shut down after this amount of time has elapsed.

or

Your workspace will automatically shut down after this amount of time has passed.

**Review by greyscaled — APPROVED** [review]
Looks great to me

**Pull request merged**